gpAux/gpdemo/generate_certs.sh

```diff
@@ -11,7 +11,7 @@ California
 Palo Alto
 Pivotal
 GPDB
-127.0.0.1
+localhost
 gpdb@127.0.0.1
```

Comment:
shouldn't we change it here as well? or it does not affect anything?

Reply:
As I understand, It is additional info. It is not used at generating or checking certificates.  It is used at info about certificate and it's owner.